core/src/main/scala/org/apache/spark/deploy/SparkSubmitArguments.scala

```diff
@@ -183,9 +183,9 @@ private[deploy] class SparkSubmitArguments(args: Seq[String], env: Map[String, S
       .orElse(sparkProperties.get(config.CORES_MAX.key))
       .orNull
     name = Option(name).orElse(sparkProperties.get("spark.app.name")).orNull
-    jars = Option(jars).orElse(sparkProperties.get("spark.jars")).orNull
-    files = Option(files).orElse(sparkProperties.get("spark.files")).orNull
-    pyFiles = Option(pyFiles).orElse(sparkProperties.get("spark.submit.pyFiles")).orNull
+    jars = Option(jars).orElse(sparkProperties.get(config.JARS.key)).orNull
+    files = Option(files).orElse(sparkProperties.get(config.FILES.key)).orNull
+    pyFiles = Option(pyFiles).orElse(sparkProperties.get(config.SUBMIT_PYTHON_FILES.key)).orNull
```

Comment:
As you are fixing "spark.submit.pyFiles" and "spark.submit.deployMode" here could you fix it in the other places as well and add it to the PR description?

Reply:
Add an extra pr?

Reply:
> As you are fixing "spark.submit.pyFiles" and "spark.submit.deployMode" here could you fix it in the other places as well and add it to the PR description?

Sorry, I don't  understand where the other places is.

1 : add an extra pr ？
2: add the description of "spark.submit.pyFiles" and "spark.submit.deployMode" in this PR?

Reply:
I have updated it. Is that what you expect?

Reply:
What I meant is keeping your change well focused. If the PR is about 'spark.jars' and 'spark.files' then I would expect only those to be modified and if you change any other config then I think it is good to add it to the title and/or description to know the reason of the change when the git log is read. Moreover in this case I would expect those touched configs to be corrected everywhere else too.

Reply:
I get it.